roles/vdm/templates/resources/platform-postgres-dataserver.yaml

```diff
@@ -0,0 +1,35 @@
+{%- set db = db_default_name_map[role] if ('database' not in settings and role in db_default_name_map) else settings.database|default(role, true) -%}
+{%- set server_name = role if role != "default" else "postgres" -%}
+apiVersion: webinfdsvr.sas.com/v1beta1
+kind: DataServer
+metadata:
+  name: {{ role }}-platform-postgres
+  annotations:
+    sas.com/default-database: "false"
+spec:
+  ssl: {{ (settings.ssl_enforcement_enabled if 'ssl_enforcement_enabled' in settings else True)|bool|lower }}
+  registrations:
+    - serviceName: {{ role }}-sas-postgres
+      dataServiceName: {{ role }}-postgres
```

Comment:
For role = cds-postgres wouldn't this result in `cds-postgres-postgres`, is this expected name?

Reply:
CDS postgres is not supported on external postgres servers. This is adding additional platform-postgres servers.
If you have a deployment directory on hand, it's based off of `/sas-bases/overlays/postgres/platform-postgres/platform-postgres-dataserver.yaml`
This mimics the behavior with Crunchy 4 with this template + multiple external PGs https://github.com/sassoftware/viya4-deployment/blob/main/roles/vdm/templates/resources/postgres-instance.yaml